packages/components/forms/src/text-input/TextInput.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: 'TextInput'
+type: 'component'
+status: 'stable'
+slug: /components/forms/src/text-input/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/forms/src/text-input'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/form-elements-textinput--basic'
+typescript: ./TextInput.tsx
+---
+
+`TextInput` is a form component, that allows users to input text or numbers and should be used in a form context.
+
+## Table of contents
+
+- [How to use TextInput](#how-to-use-TextInput)
+- [Component variations](#component-variations)
+- [Content recommendations](#content-recommendations)
+- [Accessibility](#accessibility)
+- [Props](#props)
+
+## How to use TextInput
+
+The TextInput component can be configured in a number of different ways. Use this guidance to determine the correct variation based on the context.
+
+TextInput is supporting different types of inputs, like `'text' | 'email' | 'file' | 'number' | 'password' | 'search' | 'url'`, where `text` is a default.
+Label property is always required and passed to the `<input />` as `aria-label`.
```

Comment:
I would rename it to aria-label, but I need to also do it for BaseInput I guess. Separate PR?